rust/kcl-lib/src/execution/cad_op.rs

```diff
@@ -28,6 +28,20 @@ pub enum Operation {
         is_error: bool,
     },
     #[serde(rename_all = "camelCase")]
+    VariableDeclaration {
+        /// The variable name.
+        name: String,
+        /// The value of the variable.
+        value: OpKclValue,
+        /// The visibility modifier of the variable, e.g. `export`.  `Default`
+        /// means there is no visibility modifier.
+        visibility: ItemVisibility,
```

Comment:
Is this info that we would want to show in the feature tree at some point? This is cool

Reply:
I thought it might be useful. Don't you?